samples/bluetooth/le_periph_prxp/src/prxp_rom_1_2.c

```diff
@@ -0,0 +1,189 @@
+/* Copyright (C) 2025 Alif Semiconductor - All Rights Reserved.
+ * Use, distribution and modification of this code is permitted under the
+ * terms stated in the Alif Semiconductor Software License Agreement
+ *
+ * You should have received a copy of the Alif Semiconductor Software
+ * License Agreement with this file. If not, please write to:
+ * contact@alifsemi.com, or visit: https://alifsemi.com/license
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include "alif_ble.h"
+#include "gapm_le_adv.h"
+#include "co_buf.h"
+#include "shared_control.h"
+
+/*  Profiles definitions */
+#include "llss.h"
+#include "iass.h"
+#include "tpss.h"
+#include "prxp_app.h"
+
+LOG_MODULE_REGISTER(prxp_rom_1_2, LOG_LEVEL_DBG);
```

Comment:
```suggestion
LOG_MODULE_REGISTER(prxp, LOG_LEVEL_DBG);
```
You don't compile both versions at the same time so no need to make any distinction on this level anymore.

Reply:
fixed